guides/source/active_record_callbacks.md

```diff
@@ -837,58 +837,59 @@ lead to invalid data.
 [`upsert_all`]:
     https://api.rubyonrails.org/classes/ActiveRecord/Persistence/ClassMethods.html#method-i-upsert_all
 
-Suppressing Callbacks
----------------------
+Suppressing Saving
+------------------
 
-In certain scenarios, you may need to temporarily prevent certain callbacks from
-being executed within your Rails application. This can be useful when you want
-to skip specific actions during certain operations without permanently disabling
-the callbacks.
+In certain scenarios, you may need to temporarily prevent records from being saved within your Rails application.
+This can be useful to skip the callbacks that would run if the record is saved successfully.
```

Comment:
I think the use case here is suppressing some records in complex nested callbacks:
Maybe something like:
```suggestion
In certain scenarios, you may need to temporarily prevent records from being saved within your callbacks.
This can be useful if you have a record with complex nested associations and want
to skip saving specific records during certain operations without permanently disabling
the callbacks or introducing complex conditional logic.
```